src/edu/washington/escience/myria/operator/CSVFileScanFragment.java

```diff
@@ -430,6 +441,23 @@ protected void init(final ImmutableMap<String, Object> execEnvVars) throws DbExc
                 new BufferedReader(new InputStreamReader(partitionInputStream)),
                 CSVFormat.newFormat(delimiter).withQuote(quote).withEscape(escape));
         iterator = parser.iterator();
+
+        try {
+          if (!iterator.hasNext()) {
```

Comment:
Sorry, I'm still confused. You mean the partial record has an unclosed quoted field followed by a newline at the very end of the partition?